### PR TITLE
feat(auth): handle login

### PR DESCRIPTION
**Pull request by plecrx**
> connect login to API 
> add notification provider
> notify user when error on login
> add forgotPassword ui
> member login ui
> reorder i18n
> fix prettier

### CODE REVIEW

File: frontend/src/pages/login/memberLogin/login.page.tsx [hunk 1]
```diff
@@ -1,59 +1,97 @@
...
+		const {
+			control,
+			handleSubmit,
+		} = useForm({ defaultValues: { email: "", password: "" }, mode: "onSubmit" });
+
+	const onAuthenticate = async ({ email, password }: { email: string; password: string }) => {
+		try {
+			await login(email, password);
+		} catch (error) {
+			console.log(error)
```
Comment (plecrx) [inline frontend/src/pages/login/memberLogin/login.page.tsx:28]: Je vais mettre en place un gestionnaire de notification afin que l'utilisateur soit informé en cas d'erreur.

File: frontend/src/api/auth/constants.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export const UNSAFE_COOKIE_NAME = "_auth_session_unsafe";
```
Comment (alternative-entract) [inline frontend/src/api/auth/constants.ts:1]: cookies unused !

File: frontend/src/api/auth/login.spec.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import { loginAPI } from "./login";
```
Comment (alternative-entract) [inline frontend/src/api/auth/login.spec.ts:1]: you have to set tests into CI

File: frontend/src/api/auth/logout.ts [hunk 4]
```diff
@@ -0,0 +1,3 @@
+export const logout = () => {
+	return;
+};
```
Comment (alternative-entract) [inline frontend/src/api/auth/logout.ts:3]: you are not implementing this file. 
Remove it !

File: frontend/src/api/auth/sendResetPasswordLink.ts [hunk 5]
```diff
@@ -0,0 +1,3 @@
+export const sendResetPasswordLink = () => {
+	return;
+};
```
Comment (alternative-entract) [inline frontend/src/api/auth/sendResetPasswordLink.ts:3]: you are not implementing this file.
Remove it !

File: frontend/src/api/auth/types.ts [hunk 6]
```diff
@@ -0,0 +1,34 @@
+import { UserRole } from "../../types/user";
+
+export enum ApiEndpoint {
+	REFRESH_ENDPOINT = "token/_refresh",
+	VERIFY_ENDPOINT = "token/_verify",
+	REVOKE_ENDPOINT = "token/_revoke",
+	LOGIN_ENDPOINT = "auth/login",
+	RESET_PASSWORD_ENDPOINT = "user/_reset_password",
+	UPDATE_PASSWORD_WITH_TOKEN_ENDPOINT = "_update_password_with_token",
+}
```
Comment (alternative-entract) [inline frontend/src/api/auth/types.ts:10]: please verify those endpoints with the API

File: frontend/src/api/auth/types.ts [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+export type LoginApiResponse = {
+	user?: AuthUser;
+	msg?: string;
+};
+
+export type LoginSuccess = {
+	email: string;
+	token: string;
+};
```
Comment (alternative-entract) [inline frontend/src/api/auth/types.ts:26]: rm

File: frontend/src/api/auth/types.ts [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+export enum LoginError {
+	INVALID_CREDENTIALS_USER = "Invalid Credentials user",
+	INVALID_CREDENTIALS_PASSWORD = "Invalid Credentials password",
+	BAD_REQUEST_ERROR = "Please provide email and password",
+	UNVERIFIED_EMAIL = "Please verify your email",
+	TOKEN_INVALID = "Invalid Credentials",
+}
```
Comment (alternative-entract) [inline frontend/src/api/auth/types.ts:34]: move to i18n !

File: frontend/src/api/auth/updatePasswordWithToken.ts [hunk 9]
```diff
@@ -0,0 +1,3 @@
+export const updatePasswordWithToken = () => {
+	return;
+};
```
Comment (alternative-entract) [inline frontend/src/api/auth/updatePasswordWithToken.ts:3]: rm

File: frontend/src/components/badge/badgeInfo.tsx [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+		<svg
+			width="24"
+			height="24"
+			viewBox="0 0 24 24"
+			xmlns="http://www.w3.org/2000/svg"
+		>
+			<circle cx="12" cy="12" r="12" fill="blue" />
+			<path
+				d="M12 8.99658C12.8284 8.99658 13.5 8.32501 13.5 7.49658C13.5 6.66815 12.8284 5.99658 12 5.99658C11.1716 5.99658 10.5 6.66815 10.5 7.49658C10.5 8.32501 11.1716 8.99658 12 8.99658ZM10.95 16.9536C10.95 17.5335 11.4201 18.002 12 18.0001C12.5799 17.9982 13.05 17.5265 13.05 16.9466V11.5466C13.05 10.9667 12.5799 10.4982 12 10.5001C11.4201 10.502 10.95 10.9737 10.95 11.5536V16.9536Z"
+				fill="white"
```
Comment (alternative-entract) [inline frontend/src/components/badge/badgeInfo.tsx:12]: we will have to create a palette of colors and not set hard colors...

File: frontend/src/components/badge/badgeInfo.tsx [hunk 11]
```diff
@@ -0,0 +1,16 @@
+export const BadgeInfo = () => {
+	return (
+		<svg
+			width="24"
+			height="24"
+			viewBox="0 0 24 24"
+			xmlns="http://www.w3.org/2000/svg"
+		>
+			<circle cx="12" cy="12" r="12" fill="blue" />
```
Comment (alternative-entract) [inline frontend/src/components/badge/badgeInfo.tsx:9]: we will have to create a palette of colors and not set hard colors...

**Review by alternative-entract — APPROVED** [review]

**Pull request merged**